### PR TITLE
minor updates to the install instructions

### PR DESCRIPTION
**Pull request by ocefpaf**
@MathewBiddle do you mind taking a quick look at this one? I removed some old syntax and updated a few things.

### CODE REVIEW

File: jupyterbook/content/ioos_installation_conda.md [hunk 1]
```diff
@@ -29,9 +29,9 @@ you might choose `c:\programs\Miniconda3`.
 On the "Advanced Installation Options" screen,
 leave the boxes checked if you want Miniconda 3 to be your default python.
 If you are going to be switching from Python 2 to Python 3 or perhaps some other Python distribution,
-it's best uncheck the boxes and use the `Anaconda Command Prompt` or `Anaconda Navigator` (see below for instructions) to start Anaconda.
+it is best uncheck the boxes and use the `Anaconda Command Prompt` or `Anaconda Navigator` (see below for instructions) to start Anaconda.
 
-### Linux/OS X
+### Linux/macOS
 
 Copy-and-paste this:
```
Comment (MathewBiddle) [inline jupyterbook/content/ioos_installation_conda.md:36]: should indicate that this should be ran in the terminal.

File: jupyterbook/content/ioos_installation_conda.md [hunk 2]
```diff
@@ -42,22 +42,18 @@ elif [[ $(uname) == "Linux" ]]; then
   url=https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh
 fi
 curl $url -o miniconda.sh
-bash miniconda.sh -b
+sh miniconda.sh
 export PATH=$HOME/miniconda3/bin:$PATH
 ```
 
-We also recommend to add the following line to your `~/.bashrc` file to make Miniconda the Python found first than the system Python:
+and use all the default options, expect for the license agreement where you must actively change it to `yes`.
```
Comment (MathewBiddle) [inline jupyterbook/content/ioos_installation_conda.md:49]: expect -> except

**Review by MathewBiddle — COMMENTED** [review]
This looks great! Just a few minor comments.

File: jupyterbook/content/ioos_installation_conda.md (hunk 1 above)
Comment (ocefpaf) [reply jupyterbook/content/ioos_installation_conda.md]: Good catch! We do have that below but not here.

File: jupyterbook/content/ioos_installation_conda.md (hunk 2 above)
Comment (ocefpaf) [reply jupyterbook/content/ioos_installation_conda.md]: Spellcheck is worthless when your typo exist :-)

**Pull request merged**